ord_schema/interface/query.py

```diff
@@ -72,19 +75,29 @@ def substructure_search(self,
             column = 'r'
         else:
             column = 'm'
-        do_chiral_sss = 'true' if use_stereochemistry else 'false'
-        query = f"""
-            SELECT DISTINCT A.reaction_id, A.serialized 
-            FROM reactions A 
-            INNER JOIN {table} B ON A.reaction_id = B.reaction_id
-            WHERE B.{column}@>'{pattern}'{'::qmol' if use_smarts else ''}
-            {f'LIMIT {limit}' if limit else ''};"""
-        logging.info(query)
+        components = [
+            sql.SQL("""
+                SELECT DISTINCT A.reaction_id, A.serialized 
+                FROM reactions A 
+                INNER JOIN {} B ON A.reaction_id = B.reaction_id
+                WHERE B.{}@>%s""").format(sql.Identifier(*table.split('.')),
```

Comment:
Why does `table` need to be split like this?

Reply:
In the tests, `table` comes in as something like `rdk.inputs` and using sql.Identifier(table) fails since "rdk.inputs" isn't a valid identifier but "rdk"."inputs" is --- see https://www.psycopg.org/docs/sql.html#psycopg2.sql.Identifier

Reply:
Added a comment.